nesta/core/schemas/tier_1/mappings/endpoints/health-scanner/aliases.json

```diff
@@ -1,111 +0,0 @@
-{
```

Comment:
The endpoint is unaffected, but it means that writing data to the `health-scanner` endpoint won't "be possible" (without restoring these files). The reason for dropping them is mainly for the avoidance of doubt when looking in the codebase (that `general/nih` is the place to go) and also because `health-scanner` was originally set up in ES6 so the mapping was a little different, making it more awkward to set up the hybrid "default"/"project" `mapping`

Reply:
OK thanks for the clarification, will approve now.